core/runtime/binaryen/runtime_external_interface.cpp

```diff
@@ -651,6 +653,18 @@ namespace kagome::runtime::binaryen {
         return wasm::Literal();
       }
 
+      /// ext_storage_clear_prefix_version_2
+      // TODO(Harrm): Implement limit deletion in polkadot_trie
```

Comment:
Can we create an issue for that or maybe resolve it in current PR?

Reply:
if you create an issue, please leave the link in comment

Reply:
#829

Reply:
Add it in the comment, please.

Reply:
I did it, yeah